site/content/blog/markdowndb-launch.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Announcing the official launch of MarkdownDB! 🚀
+description: We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website at markdowndb.com
+date: 2023-10-11
+authors: ['Ola Rubaj']
+filetype: 'blog'
+---
+
+Hello, dear readers!
+
+We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website!
+
+Visit https://markdowndb.com/
+
+## 🔍 What is MarkdownDB?
+
+MarkdownDB transforms your Markdown content into a queryable, lightweight SQLite database. Imagine being able to treat your collection of markdown files like entries in a database! Ever thought about fetching documents with specific tags or created in the past week? Or, say, pulling up all tasks across documents marked with the "⏭️" emoji? With MarkdownDB, all of this (and more) is not just possible, but also a breeze.
+
+## 🌟 Features
+
+- Rich metadata extracted including frontmatter, links and more.
+- Lightweight and fast indexing 1000s of files in seconds.
+- Open source and extensible via plugin system.
+
+## 🚀 Quick start
+
+1. Start with a collection of markdown files. These could be your blog posts, project docs, or any other type of content. Each file can be enhanced with a YAML frontmatter to include metadata.
+
+2. Index your files with MarkdownDB. Use our npm `mddb` package to index your Markdown files into an SQLite database.
+
+3. And that's it! Now you're all set to query your files with SQL or integrate with the MarkdownDB Node.js API for a more customized experience.
+
+👉 [Read the full tutorial](https://markdowndb.com/blog/basic-tutorial)
+
+---
+
+Explore the world of MarkdownDB on [our new official website](https://markdowndb.com/) and support us on [GitHub](https://github.com/datopian/markdowndb).
```

Comment:
suggest

```
Find out more on our new official website: https://markdowndb.com/

Check out the source on GitHub https://github.com/datopian/markdowndb
```

(Why: where links are informative and useful i think it is worth including them in full as they are here - then i know where i am going if i click on them etc)